tensionmap.py

```diff
@@ -43,6 +44,23 @@
 
 tm_update_modes = ["OBJECT", "WEIGHT_PAINT", "VERTEX_PAINT"]
 
+class Object_Data(bpy.types.Object):
+    def __init__(self,obj):
+        self.original_edge_lengths = self.__calculate_original_edge_lengths(obj)
+        
+    def __calculate_original_edge_lengths(self, obj):
+        original_edge_lengths = [0]*len(obj.data.edges)
+        for i in range(len(obj.data.edges)):
+            edge = obj.data.edges[i]
+            first_vertex = edge.vertices[0]
+            second_vertex = edge.vertices[1]
+            original_edge_lengths[i] = (obj.data.vertices[first_vertex].co -
+                                    obj.data.vertices[second_vertex].co).length
+        return original_edge_lengths
+
+    @property
+    def original_edge_length(self): 
```

Comment:
I'm afraid I don't really follow the logic here. It doesn't help that there is  member called `original_edge_length**s**` with an s, and a method tagged as property named `original_edge_length` without s.

Reply:
Oh, the property is a typo, it should be `original_edge_lengths` in all cases.
The __calculate_original_edge_lengths function is just a helper function for initialization, but I guess the return value is superfluous, since we have access to self and could fill the variables inside the function, right?
As I mentioned, I am new to Python, so if something is not Pythonian, feel free to make suggestions!
PS: GitHub seems to have a "Insert a suggestion" feature, maybe we can try that here?

Reply:
I'll delay fixing this for now due to your other comments.

Reply:
Maybe that object could be replaced with a named tuple. See https://stackoverflow.com/a/36033
And the two init functions for each values would become their own pure function, called when creating a new named tuple. (I like pure functions).